controllers/admin/AdminImportController.php

```diff
@@ -1428,32 +1992,34 @@ protected function categoryImportOne($info, $default_language_id, $id_lang, $for
         //copying images of categories
         if (isset($category->image) && !empty($category->image)) {
             if (!(AdminImportController::copyImg($category->id, null, $category->image, 'categories', !$regenerate))) {
-                $this->warnings[] = $category->image.' '.$this->trans('cannot be copied.', array(), 'Admin.Advparameters.Notification');
+                $this->warnings[] = $category->image.' '.$this->trans('cannot be copied.',
+                    array(),
+                    'Admin.Advparameters.Notification'
+                );
             }
         }
         // If both failed, mysql error
         if (!$res) {
-            $this->errors[] = $this->trans(
-                '%1$s (ID: %2$s) cannot be %3$s',
-                array(
-                    (isset($info['name']) && !empty($info['name']))? Tools::safeOutput($info['name']) : 'No Name',
-                    (isset($info['id']) && !empty($info['id']))? Tools::safeOutput($info['id']) : 'No ID',
-                    ($validateOnly?'validated':'saved')
-                ),
-                'Admin.Advparameters.Notification'
-            );
-            $error_tmp = ($field_error !== true ? $field_error : '').(isset($lang_field_error) && $lang_field_error !== true ? $lang_field_error : '').Db::getInstance()->getMsgError();
-            if ($error_tmp != '') {
-                $this->errors[] = $error_tmp;
+            $this->errors[] = $this->trans('%1$s (ID: %2$s) cannot be %3$s', array(
+                !empty($info['name']) ? Tools::safeOutput($info['name']) : 'No Name',
+                !empty($categoryId) ? Tools::safeOutput($categoryId) : 'No ID',
+                $validateOnly ? 'validated' : 'saved',
+            ), 'Admin.Advparameters.Notification');
+            $errorTmp = ($fieldError !== true ? $fieldError : '')
+                . (isset($langFieldError) && $langFieldError !== true ? $langFieldError : '')
+                . Db::getInstance()->getMsgError();
+            if ($errorTmp != '') {
+                $this->errors[] = $errorTmp;
             }
         } else {
             // Associate category to shop
-            if ($shop_is_feature_active) {
-                Db::getInstance()->execute('
-					DELETE FROM '._DB_PREFIX_.'category_shop
-					WHERE id_category = '.(int)$category->id);
+            if ($shopIsFeatureActive) {
+                Db::getInstance()->execute(
+                    'DELETE FROM '._DB_PREFIX_.'category_shop'
+                    . ' WHERE id_category = '.(int)$category->id
+                );
 
-                if (!$shop_is_feature_active) {
+                if (!$shopIsFeatureActive) {
```

Comment:
Never true, can be removed

Reply:
So, it looks like I should remove the whole block from line 2015 to line 2038. Am I right ?

Reply:
Will do.